app/lib/email_sender.dart

```diff
@@ -0,0 +1,74 @@
+import 'package:flutter_email_sender/flutter_email_sender.dart';
+import 'package:flutter/material.dart';
+
+class EmailSender{
+
+  Future<void> send(String emailBody, String eSubject, String eRecipients) async{
+    final Email email = Email(
+      body: emailBody,
+      subject: eSubject,
+      recipients: [eRecipients],
+      attachmentPaths: null,
+      isHTML: false,
+    );
+    String emailResponse;
+    try{
+      await FlutterEmailSender.send(email);
+      emailResponse = 'success';
+    }catch(error){
+      emailResponse = error.toString();
+    }
+  }
+}
+
+class EmailRoute extends StatelessWidget{
+  final formKey = GlobalKey<FormState>();
+
+  final email = EmailSender();
+  final recipientController = TextEditingController();
+  final subjectController = TextEditingController();
+  final bodyController = TextEditingController();
+
+  Widget build(BuildContext context){
+    return Scaffold(
+        appBar: AppBar(
+          title: Text('Email Form'),
+        ),
+        body: Padding(
+            padding: const EdgeInsets.all(16.0),
+            child: Form(
+              key: formKey,
+              child: Column(children: <Widget>[
+                TextFormField(
+                  decoration: InputDecoration(hintText: 'Email'),
+                  controller: recipientController,
+                  validator: (value){
+                    if(value.isEmpty){
+                      return 'Please enter email';
+                    }else
+                      return null;
```

Comment:
We want to avoid using null in Dart (as with most languages, Java had originally introduced it's abuse and it causes bugs all the time...) see [here for options on how we could refactor this to be non-null or at least safe](https://dart.dev/null-safety).

- [ ] Make this null-safe if possible

Reply:
I know that this commit is to explore email sending feature, but do note that there should be only one class in one file. In the future (no need to change), EmailSender can go to model directory and email route can go to navigations directory.

Reply:
changed form validation to null safe expression